src/main/java/pl/piekoszek/gorskimatches/facebook/FacebookMessageService.java

```diff
@@ -4,22 +4,86 @@
 import org.springframework.http.HttpEntity;
 import org.springframework.stereotype.Component;
 import org.springframework.web.client.RestTemplate;
+import pl.piekoszek.gorskimatches.equation.EquationRandomizer;
+import pl.piekoszek.gorskimatches.equation.QuizAnswerChecker;
+import pl.piekoszek.gorskimatches.validation.StringEditor;
+
+import java.util.HashMap;
+import java.util.Map;
 
 @Component
 public class FacebookMessageService {
```

Comment:
This class has too many responsibilities, for example handling commands could be done by another component. Also preparing entities to send via HTTP could be separated.